events/events_test.go

```diff
@@ -137,19 +147,19 @@ func TestFanout1(t *testing.T) {
 							break listenLoop
 						} else {
 							if sdata.x != x+1 {
-								log.Fatalf("Did not get correct data in event.\n")
+								log.Fatalf("Did not get correct data in event.")
```

Comment:
This is removing all the new line chars

Reply:
Ah yes, I think its the rebase, I'll re-push.

Reply:
@SenRamakri Since I made an additional commit, I think you'll need to do a `git pull` before you do the rebase against the armPelionEdge master branch.